test/unit/blacklister-test.js

```diff
@@ -0,0 +1,24 @@
+const assert = require('assert')
+const Blacklister = require('../../app/scripts/blacklister')
```

Comment:
I usually like to reserve capitalized variables for class names, and name functions after what they do, so this might be `shouldBlacklist`.